tensorflow/core/kernels/cudnn_rnn_ops.cc

```diff
@@ -502,20 +502,23 @@ struct CudnnRnnModelShapes {
   int dir_count;
   int max_seq_length;
   int batch_size;
+  int c_num_units;
```

Comment:
Please change all `c_` to `cell_` to make it easier to read (avoiding acronyms) and make the names consistent within the context. E.g. use `c_` if it used `h_` in the context, and `cell_` if it used `hidden_`.